apps/tutorial/public/docs/5-builtins/3-htmlSafe/prose.md

```diff
@@ -1,6 +1,6 @@
 Glimmer, by default, protects you against [Cross-Site-Scripting][xss] attacks, where an attack may try to get malicious code rendered on a page to scrape or send information from other user sessions.
 
-The `htmlSafe` utility is used when you have input that either _know_ to be safe, or you can be reasonably certain that rendering untrusted input will have no negative consequence (for example, maybe untrusted input is ran through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify)).
+The `htmlSafe` utility is used when you have an input that you either _know_ to be safe or are reasonably certain that rendering any malicious input will have no negative consequence (for example, maybe the untrusted input is already safely handled by running it through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify -- making the input trusted and safe)). `htmlSafe` does not make the string safe; in fact, it tells Handlebars to ignore the typical step of making the string safe. Use it only when you do not want the input to be HTML escaped.
```

Comment:
```suggestion
The `htmlSafe` utility is used when you have an input that you either _know_ to be safe or are reasonably certain that rendering any malicious input will have no negative consequence (for example, maybe the untrusted input is already safely handled by running it through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify -- making the input trusted and safe)). `htmlSafe` does not make the string safe; in fact, it tells Ember's renderer to ignore the typical step of making the string safe. Use it only when you do not want the input to be HTML escaped.
```

We don't actually use handlebars :sweat_smile:

Reply:
Ember's renderer = glimmer-vm, but we don't need go that deep, because then we'd need to explain it :sweat_smile: